javascript/javascript1/week4/Ali-Nematollahi-Homework/high-and-low.js

```diff
@@ -0,0 +1,11 @@
+function highAndLow(numbers) {
+  const stringArray = numbers.split(" ");
+  const numbersArray = [];
+  for (i in stringArray) {
```

Comment:
```suggestion
  for (let i in stringArray) {
```
Same comment as the other file.